pkg/utils/helpers.go

```diff
@@ -25,83 +27,86 @@ var (
 	}
 )
 
-func GetOceanCdEntityKindByName(entityType string) (string, error) {
-	if entityType == "cluster" || entityType == "Cluster" || entityType == "clusters" || entityType == "Clusters" {
-		return "", errors.New(fmt.Sprintf("error: Unrecognize resource type '%s'", entityType))
-	}
-
-	return GetEntityKindByName(entityType)
+type ConfigHandler interface {
```

Comment:
I didn't see the size of this file and I think we should split it multiple helpers files by type
json_conffiguration_helpers
yaml_conffiguration_helpers
etc
wdyt?

Reply:
I guess the configuration_hendlers file will be enough
ok?